jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -438,7 +438,7 @@ def step900_collect_driver_artifacts() {
 def create_params_file(archive_dir) {
     def keys = params.keySet()
     def param_string = ""
-    for (i = 0; i < keys.length; i++) {
+    for (i = 0; i < keys.size(); i++) {
```

Comment:
For people who don't know these things, (like me until just now!), it might be helpful to note in the commit that length returns the capacity of an array and size returns the number of elements. If I'm reading things right.

Reply:
Oh is that it? The error I got in Jenkins was rather cryptic:
```
org.jenkinsci.plugins.scriptsecurity.sandbox.RejectedAccessException: unclassified field java.util.Collections$UnmodifiableSet length
```
I just happened to have seen `length` and `size()` described as interchangeable by the Groovy docs on Arrays http://grails.asia/groovy-array-manipulation-examples.

Reply:
Guess that's what I get for using not-real-docs.

Reply:
Oh look, I found a better thing:
```
map.each { entry ->
    println "Name: $entry.key Age: $entry.value"
}
```

Reply:
Ah yeah that's at least the case for Java so since it was equally confusing I thought maybe it would be the same here (idk how Groovy interop works, especially now in the Jenkins world)